### PR TITLE
Adding Akka Scala sample to the Azure Event Hubs repository

### PR DESCRIPTION
**Pull request by kmansel**
This Scala code is a sample on how to get started with Akka -> Kafka with Event Hubs on Azure

### CODE REVIEW

File: samples/kafka/akka/scala/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Using Akka Streams with Event Hubs for Apache Kafka Ecosystems (Scala)
+
+One of the key benefits of using Apache Kafka is the number of ecosystems it can connect to. Kafka-enabled Event Hubs allow users to combine the flexibility of the Kafka ecosystem with the scalability, consistency, and support of the Azure ecosystem without having to manage on prem clusters or resources - it's the best of both worlds!
+
+This tutorial will show how to connect Akka Streams to Kafka-enabled Event Hubs without changing your protocol clients or running your own clusters. Azure Event Hubs for Apache Kafka Ecosystems supports [Apache Kafka version 1.0.](https://kafka.apache.org/10/documentation.html)
+
+## Prerequisites
+
+If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/en-us/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio) before you begin.
```
Comment (basilhariri) [inline samples/kafka/akka/scala/README.md]: Remove "en-us/" from link

File: samples/kafka/akka/scala/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    * On Ubuntu, run `apt-get install default-jdk` to install the JDK.
+    * Be sure to set the JAVA_HOME environment variable to point to the folder where the JDK is installed.
+* [sbt](https://www.scala-sbt.org/download.html)
+    * On Ubuntu, follow the directions at [https://www.scala-sbt.org/release/docs/Installing-sbt-on-Linux.html](https://www.scala-sbt.org/release/docs/Installing-sbt-on-Linux.html)
+* [Git](https://www.git-scm.com/downloads)
+    * On Ubuntu, you can run `sudo apt-get install git` to install Git.
+
+## Create an Event Hubs namespace
+
+An Event Hubs namespace is required to send or receive from any Event Hubs service. See [Create Kafka-enabled Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create-kafka-enabled) for instructions on getting an Event Hubs Kafka endpoint. Make sure to copy the Event Hubs connection string for later use.
```
Comment (basilhariri) [inline samples/kafka/akka/scala/README.md]: Remove "en-us/" from link

File: samples/kafka/akka/scala/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+## Clone the example project
+
+Now that you have a Kafka-enabled Event Hubs connection string, clone the Azure Event Hubs repository and navigate to the `samples/kafka/akka/scala` subfolder:
+
+```bash
+git clone https://github.com/Azure/azure-event-hubs.git
+cd azure-event-hubs/samples/kafka/akka/scala
```
Comment (basilhariri) [inline samples/kafka/akka/scala/README.md]: cd azure-event-hubs/samples/kafka/akka-scala

File: samples/kafka/akka/scala/producer/src/main/scala/ProducerMain.scala [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    // topic to send the message to on event hubs
+    val topic = "test"
+
+    // loop until the program is exited, send message every 2 seconds
+    Source.repeat(0)
+      .throttle(1, FiniteDuration(2, SECONDS), 1, ThrottleMode.Shaping)
+      .map(_ => {
+        //construct our message here and grab a random name from our names list
+        val message = s"Welcome to Azure Event Hubs ${namesList(Random.nextInt(namesList.size))}!"
```
Comment (basilhariri) [inline samples/kafka/akka/scala/producer/src/main/scala/ProducerMain.scala]: send a message like "Akka Scala Producer Message #X" where X ranges from 1 - 100

**Comment by kmansel** [comment]
Closing PR, will add another here with requested changes and directory structure updates.

**Pull request closed without merging**